src/main.cpp

```diff
@@ -169,14 +169,9 @@ QCoreApplication* createApplication(int &argc, char *argv[])
 void myMessageOutput(QtMsgType type, const QMessageLogContext &context, const QString &msg)
 {
     QSettings settings;
-    static bool debuglog = settings.value("log_debug", false).toBool();
-    if(debuglog == false)
-#if defined(Q_OS_LINUX) || defined(Q_OS_WINDOWS)
-#ifndef Q_OS_ANDROID
-        if(forceQml)
-#endif
-#endif
-        return;
+    static bool logdebug = settings.value("log_debug", false).toBool();
+    if(nologs == true && logdebug == false)
```

Comment:
> the log on the screen ok, but not on the disk.

You're right ! 

But if we run the qz as a service (like I do for headless client) these logs to stdout are going to system logs.
This could be arranged with 2>/dev/null, but this isn't that great.

I think I found a workaround that satisfies every operating system (see PR updated)

Reply:
yeah i like it. Did you already try in your setup?